<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>391</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>670</width>
    <height>391</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>670</width>
    <height>391</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>CFL Creator &amp; Converter</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="cflFilesList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>671</width>
      <height>371</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>671</width>
      <height>371</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>671</width>
      <height>371</height>
     </size>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Filename</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>File Size</string>
     </property>
     <property name="textAlignment">
      <set>AlignHCenter|AlignVCenter|AlignCenter</set>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen_CFL">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_CFL"/>
    <addaction name="actionCreate_CFL"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionExtract_Single_File"/>
    <addaction name="separator"/>
    <addaction name="actionExtract_All"/>
    <addaction name="actionConvert_to_CHKN"/>
   </widget>
   <addaction name="menuOpen_CFL"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionExtractSingle"/>
   <addaction name="actionExtract"/>
   <addaction name="separator"/>
   <addaction name="actionNew"/>
  </widget>
  <action name="actionOpen_CFL">
   <property name="text">
    <string>Open CFL</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionExtract_All">
   <property name="text">
    <string>Extract All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+E</string>
   </property>
  </action>
  <action name="actionConvert_to_CHKN">
   <property name="text">
    <string>Convert to CHKN</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+C</string>
   </property>
  </action>
  <action name="actionCreate_CFL">
   <property name="text">
    <string>Create CFL</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open CFL</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New CFL</string>
   </property>
  </action>
  <action name="actionExtract">
   <property name="text">
    <string>Extract All Files</string>
   </property>
  </action>
  <action name="actionExtract_Single_File">
   <property name="text">
    <string>Extract Single File</string>
   </property>
  </action>
  <action name="actionExtractSingle">
   <property name="text">
    <string>Extract Single File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
